dictionaries/hr_HR/cspell-ext.json

```diff
@@ -23,7 +23,7 @@
             "languageId": "*",
             // Language local. i.e. en-US, de-AT, or ru. * will match all locals.
             // Multiple locals can be specified like: "en, en-US" to match both English and English US.
-            "local": "hr,hr-HR",
+            "local": "hr",
```

Comment:
It is best to keep this as `hr,hr-HR` that way the dictionary can be used with either locale setting.